src/pages/components/tile/usage.mdx

```diff
@@ -399,13 +424,14 @@ varying images sizes.
 
 ### Alignment
 
-Always strive for left alignment whenver possible. Type in tiles should always
-be left aligned. Icons or icon buttons can be aligned to either left or right
-edge depending on the tiles’ complexity.
+#### Icon and call-to-action button
+
+With clicktable tiles, place the icon at the bottom right to indicate
+navigation. Use the interactive token for the icon. Avoid using interactive link
+that can be confusing as multiple click targets.
 
-For example: In situations where there are both an icon or pictogram with an
-icon button, place the icon or pictorgram in the lower left corner of the tile
-and move the icon button (action) to the right corner.
+With non-clickable tiles(or base tile), place the interactive components, such
+as button or link, at the bottom left to indicate call-to-action.
```

Comment:
should there be other images under this section? it's calling for either bottom right or bottom left alignment but the pictures are showing top right
<img width="835" alt="image" src="https://user-images.githubusercontent.com/40550942/214892566-ab8ec8f1-59f2-458e-961f-8737628a51a1.png">